src/app.js

```diff
@@ -78,6 +78,7 @@ var app = function(options) {
          options.env == 'production',       'env must be production or development');
   assert(options.forceSSL !== undefined,    'forceSSL must be defined');
   assert(options.trustProxy !== undefined,  'trustProxy must be defined');
+  assert(options.docs, 'options.docs must be given');
```

Comment:
Are we sure we intend on making this required especially since we check below if `rootDocsRedirect` is defined before using it? What if we instead

```js
assert(!options.rootDocsRedirect || options.docs, 'options.docs must be given if rootDocsRedirect is specified');
```